src/main/java/com/codeninjas/productservice/controller/ProductController.java

```diff
@@ -20,7 +21,7 @@ public class ProductController {
 
     @PutMapping(path = "/{id}")
     @ResponseStatus(HttpStatus.OK)
-    public ResponseEntity<?> updateById(@PathVariable String id, @RequestBody Product product) {
+    public ResponseEntity<?> updateById(@PathVariable String id, @Valid @RequestBody Product product) throws ProductServiceException {
         productService.updateProduct(id ,product);
         return new ResponseEntity<String>("Updated",HttpStatus.OK);
```

Comment:
Remove message and return status only
return new ResponseEntity<String>(HttpStatus.OK);

Reply:
done!